### PR TITLE
fix test

### PR DESCRIPTION
**Pull request by Yostra**

### CODE REVIEW

File: tests/core/full_node/test_full_node.py [hunk 1]
```diff
@@ -665,7 +666,7 @@ async def test_request_sub_blocks(self, wallet_nodes):
         )
         assert len(res.data.sub_blocks) == 6
         assert res.data.sub_blocks[-1].transactions_generator is not None
-        assert res.data.sub_blocks[-1] == blocks[-1]
+        assert std_hash(res.data.sub_blocks[-1]) == std_hash(blocks_t[-1])
```
Comment (arvidn) [inline tests/core/full_node/test_full_node.py:669]: do subblocks not come with comparison operators?

File: tests/core/full_node/test_full_node.py [hunk 2]
```diff
@@ -613,7 +613,7 @@ async def test_request_sub_blocks(self, wallet_nodes):
         blocks = await full_node_1.get_all_full_blocks()
 
         blocks = bt.get_consecutive_blocks(
-            30,
+            33,
```
Comment (arvidn) [inline tests/core/full_node/test_full_node.py:616]: As someone new to this codebase, it's really hard to know the significance of this number. I think there should be a comment here, explaining why 33 was chosen here, maybe even with a reference to the limit that I think you mentioned earlier.

File: tests/core/full_node/test_full_node.py (hunk 1 above)
Comment (Yostra) [reply tests/core/full_node/test_full_node.py:669]: nope

**Review by arvidn — APPROVED** [review]

File: tests/core/full_node/test_full_node.py (hunk 1 above)
Comment (mariano54) [reply tests/core/full_node/test_full_node.py:670]: We should use .header_hash or .get_hash() instead of std_hash though

**Pull request merged**